pkg/plugins/policies/core/rules/rules.go

```diff
@@ -179,6 +233,40 @@ func isSubset(t1, t2 Tag) bool {
 
 // Intersect returns true if there exists an element that belongs both to 'other' and current set.
 // Empty set intersects with all sets.
+//
+// We're using this function to check if 2 'from' rules of MeshTrafficPermission can be applied to the same client DPP.
+// For example:
+//
+/*
+from:
+  - targetRef:
+      kind: MeshSubset
+      tags:
+        team: team-a
+  - targetRef:
+      kind: MeshSubset
+      tags:
+        zone: east
+*/
+//
+// there is a DPP with tags 'team: team-a' and 'zone: east' that's subjected to both these rules.
+// So 'from[0]' and 'from[1]' have an intersection.
+// However, in another example:
+//
+/*
+from:
+  - targetRef:
+      kind: MeshSubset
+      tags:
+        team: team-a
+  - targetRef:
+      kind: MeshSubset
+      tags:
+        team: team-b
+        zone: east
+*/
```

Comment:
can we use single line `//` comment on all lines here? multiline comment looks a bit odd when combined with single line comments

Reply:
That's odd, `make check` works for me as expected with `//`. If you don't mind I can push my change to your branch. Maybe your IDE somehow changes the indentation